src/modality-agnostic-files/data-summary-files.md

```diff
@@ -132,6 +132,177 @@ It is RECOMMENDED to accompany each `samples.tsv` file with a sidecar
 }
 ```
 
+## Phenotypic and assessment data
+
+Template:
+
+```Text
+phenotype/
+    <measurement_tool_name>.tsv
+    <measurement_tool_name>.json
```

Comment:
I can't remember if we were set on using snake_case naming for tool names? Do we need to specify it somewhere (or is it already?)